README.md

```diff
@@ -1 +1,40 @@
-# newyork-times-clone
\ No newline at end of file
+# New York Times Webpage Clone
+
+This is a copy of the New York Webpage.
+
+![screenshot](https://github.com/shubhsk88/newyork-times-clone/blob/clone/img/screenshot2.png)
+
+## Built With
+
+- HTML,
+- CSS,
+- Used fontawesome.com for the icons.
+
+## Live Demo
+
+
+ [Live Demo](https://rawcdn.githack.com/shubhsk88/newyork-times-clone/1d0fc557878baa9914b92ae7c4c9916a7b2d4393/index.html)
+
+
+## Authors
+
+👤 **Mauricio Fow**
+
+👤 **Shubham Singh**
+
```

Comment:
- [ ] Add information about you! Include your LinkedIn, Twitter or other social media so people can contact you!